android/engine/src/main/java/org/smartregister/fhircore/engine/ui/questionnaire/QuestionnaireActivity.kt

```diff
@@ -137,7 +137,7 @@ open class QuestionnaireActivity : BaseMultiLanguageActivity(), View.OnClickList
           text = context.getString(R.string.done)
         } else if (editMode) {
           text = getString(R.string.edit)
-        }
+        } else if (questionnaire.experimental) text = context.getString(R.string.done)
```

Comment:
can something be experimental and in `editMode`? if not you can move this into the conditional on line 136